lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -105,6 +1959,10 @@
             __defaultLocale = locale;
         }
 
+        if (isPlatformUsingWinGlob) {
```

Comment:
Is it worth it going through and removing these isPlatformUsingWinGlob checks, since the code here and below is only run if thats true? The downside is that the diff gets more complex and this code will likely be going away soon regardless.

Reply:
Up to you. The diff will either come up now or later. Since this change is new features, I'd vote to let this be. A follow up can be made to remove old/dead code at the appropriate time.